### ISSUE TITLE
 Unskip Admin shopping list test incompatible with A11y in 6.6.x

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#8078]
Admin shopping list in 6.6.x are incompatible with A11y. This is probably due to a missing backport PR since they’re passing in the trunk. 
They skipped in this [PR|https://github.com/shopware/SwagCommercial/pull/239/files].

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40785**

 Jira Issue URL: [https://shopware.atlassian.net/browse/NEXT-40785](https://shopware.atlassian.net/browse/NEXT-40785)



[created from NEXT-40785, comment 548752]

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40785**



[created from NEXT-40785, comment 548753]
